util/libcrypto.num

```diff
@@ -5438,3 +5438,5 @@ ASYNC_set_mem_functions                 ?	3_1_0	EXIST::FUNCTION:
 ASYNC_get_mem_functions                 ?	3_1_0	EXIST::FUNCTION:
 BIO_ADDR_dup                            ?	3_1_0	EXIST::FUNCTION:SOCK
 CMS_final_digest                        ?	3_1_0	EXIST::FUNCTION:CMS
+OPENSSL_strcasecmp                      ?	3_1_0	EXIST::FUNCTION:
+OPENSSL_strncasecmp                     ?	3_1_0	EXIST::FUNCTION:
```

Comment:
The version here should be 3_0_3 if we make it public in 3.0.3.

Reply:
Fixed, thanks!